### PR TITLE
Cart styling

### PR DESCRIPTION
**Pull request by areebali99**
### Goal
styling improved after getting merged 

### work
styling improved after getting merged

### CODE REVIEW

File: src/components/Drawer/index.js [hunk 1]
```diff
@@ -57,44 +61,48 @@ const TemporaryDrawer = () => {
...
+                      return (
+                        <DrawerCard key={cartdata.id}>
+                          <AddToCartImg alt="cart" src={cartdata.img} />
 
-                        <DrawerPrice>
-                          <Add>
-                            <div>
-                              <h4>{cartdata.name}</h4>
+                          <DrawerPrice>
+                            <div style={{ position: 'relative', marginLeft: '5vh' }}>
```
Comment (SiddiquiMonis401) [inline src/components/Drawer/index.js:71]: Remove inline styles

File: src/components/Drawer/style.js [hunk 2]
```diff
@@ -139,3 +151,9 @@ export const ModalIcons = styled.div`
 export const PriceSpan = styled.span`
   margin: 20px;
 `;
+export const EmptyCartHeading = styled.h2`
```
Comment (SiddiquiMonis401) [inline src/components/Drawer/style.js:154]: Use typography here!!

**Review by SiddiquiMonis401 — CHANGES_REQUESTED** [review]

File: src/Features/Customer/Dashboard/index.js [hunk 3]
```diff
@@ -1,47 +1,21 @@
 import React from 'react';
 
-import { Grid, Typography, Box } from '@material-ui/core';
-import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import { Box } from '@material-ui/core';
+import { Grid } from '@material-ui/core';
```
Comment (ShahirZain) [inline src/Features/Customer/Dashboard/index.js:4]: we can do this in single line

File: src/Features/Customer/ducks.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-case-declarations */
```
Comment (ShahirZain) [inline src/Features/Customer/ducks.js:1]: remove

**Review by ShahirZain — CHANGES_REQUESTED** [review]

**Review by SiddiquiMonis401 — APPROVED** [review]

**Review by ShahirZain — APPROVED** [review]

**Pull request merged**